projects/cuelang.org/package.yml

```diff
@@ -0,0 +1,26 @@
+distributable:
+  url: https://github.com/cue-lang/cue/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: cue-lange/cue
+
+provides:
+  - bin/cue
+
+build:
+  script: |
+    go mod download
+    go build -v -ldflags="$LDFLAGS" -o "{{ prefix }}"/bin/cue ./cmd/cue
```

Comment:
There is directory in the root of this repo named `cue`. If I try to build locally without specifying an output file I receive the following error.

```sh
go: build output "cue" already exists and is a directory
```

Reply:
i fixed a bug in the self-hosted (aarch64) runners caused by go mod leaving unwritable files behind, which prevented cleanup; hopefully this is fixed now.